ndspy/bmg.py

```diff
@@ -21,7 +21,10 @@
 
 import os
 import struct
-from typing import Literal
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Literal
```

Comment:
This trick here is to get around the fact that `typing.Literal` isn't available in Python 3.7 (it was introduced in 3.8). I _think_ the "proper" way to do this is to install `typing_extensions` and import from there on 3.7. This seems to work just as well and appears to pass `mypy`, and avoids adding `typing_extensions` as a dependency. Let me know if you have any thoughts or would like me to change this to something else.

Reply:
This looks much better to me than adding a dependency. ndspy currently has zero dependencies, and I think that's a nice property to preserve until there's a compelling reason to add one.